### DISCUSSION TITLE
内网绕过

### DISCUSSION THREAD

**Discussion by Lanlan13-14** [vernesong/OpenClash#4136] [General]
根据官方文档在Fakeip下设置了192.168.0.20-192.168.0.254的IP段走Direct之后这个IP段内的设备无法访问外网只能访问中国大陆的网站，希望修复一下这个bug

**Comment by vernesong** [comment]
直连不就是这样的吗

**Comment by refined-fish** [comment]
不是你自己说设的Direct吗，那Direct，不就只能访问大陆吗
![7fc0b4cec0c56e0d37e2ac560de89c44](https://github.com/user-attachments/assets/500b7415-4d9d-4b70-a995-00992900cd79)
